blocks/related-blogs/related-blogs.js

```diff
@@ -1,12 +1,116 @@
 import { readBlockConfig } from '../../scripts/lib-franklin.js';
 
 function getConfiguredTag(config) {
-  // should we have a default if no tag configured?
-  return config.tags;
+  // should we have a default if no tag is configured?
+  return config.tag;
 }
+
+async function enrichProfiles(rArticles) {
+  const response = await fetch('/profile-blog.json');
+  const json = await response.json();
+  // console.debug(JSON.stringify(json.data));
+
+  Object.entries(rArticles).forEach((entry) => {
+    // eslint-disable-next-line no-unused-vars
+    const [key, value] = entry;
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
Why is this for not allowed? If it's not useful, can we just disable that linter setting?

Reply:
fixed / removed